src/server/apis/dash-data.ts

```diff
@@ -197,6 +202,15 @@ export async function fetchIncomingData(
       reviewedPr.status = {type: 'ApprovalRequired'};
     }
 
+    const lastActivity = await getPRLastActivityTimestamp(reviewedPr);
+    if (lastActivity) {
+      reviewedPr.hasNewActivity = await issueHasNewActivity(
+          dashboardLogin, lastActivity, lastViewedInfo[pr.id]);
```

Comment:
This is in the loop, so don't want to be doing more reads than necessary. See comment in `issuesHasNewActivity()`.